update screenshots (#9856)
fishtown-analytics/docs.getdbt.com: website/snippets/_use-copilot-resources.md

Reviewer: Hiya Mirna

I had a look in my account and there is no **dbt Copilot** button and no **Generate Metrics** option.

I also checked `emptyStatePrompts.ts` in dbt-ui. With a `__semantic_models` YAML file open, Wizard shows **Help me add metrics**.

Maybe replace it with something like:

- Once defined, open the semantic models YAML file, then in the **<Constant name="wizard" />** tab click **Help me add metrics**.

Author: the help me add metrics is for wizard i believe

Generate documentation, tests, metrics, and semantic models [resources](/docs/build/projects) with the click-of-a-button in the [<Constant name="studio_ide" />](/docs/platform/studio-ide/develop-in-studio) using dbt Copilot, saving you time. To access and use this AI feature:

1. Navigate to the <Constant name="studio_ide" /> and select a SQL model file under the **File Explorer**.
2. In the **Console** section (under the **File Editor**), click the dbt Copilot button and choose what you want to generate, such as documentation, tests, metrics, SQL, or semantic models. To generate multiple YAML configs for the same model, select each option separately. dbt Copilot saves them to the same YAML file.

   <CopilotLimitation />

   - To generate metrics, you need to first have semantic models defined. 

>    - Once defined, click **dbt Copilot** and select **Generate Metrics**.

Author: ```suggestion
   - Once defined, click **dbt Copilot** and select **Generate metrics**.
```

Generate documentation, tests, metrics, and semantic models [resources](/docs/build/projects) with the click-of-a-button in the [<Constant name="studio_ide" />](/docs/platform/studio-ide/develop-in-studio) using dbt Copilot, saving you time. To access and use this AI feature:

1. Navigate to the <Constant name="studio_ide" /> and select a SQL model file under the **File Explorer**.

> 2. In the **Console** section (under the **File Editor**), click the AI option to view the available AI options.

Author: ```suggestion
2. In the **Console** section (under the **File Editor**), click the AI option to view the available AI actions.
```

> 2. In the **Console** section (under the **File Editor**), click the AI option to view the available AI actions.

Reviewer: Hiya Mirna

I wasn't sure if the AI option was only available to some customers or named something else, but I couldn't see an AI option in my acct. 

<img width="1433" height="788" alt="Image" src="https://github.com/user-attachments/assets/33d8ef38-bcd4-483b-b5be-5e50b558b146" />

Author: responded in slack

> 2. In the **Console** section (under the **File Editor**), click the AI option to view the available AI actions.

Author: ```suggestion
2. In the **Console** section (under the **File Editor**), click **dbt Copilot** to view the available AI actions.
```

> 2. In the **Console** section (under the **File Editor**), click **dbt Copilot** to view the available AI actions.

Author: ```suggestion
2. In the **Console** section (under the **File Editor**), click the dbt Copilot button and choose what you want to generate, such as documentation, tests, metrics, SQL, or semantic models. To generate multiple YAML configs for the same model, select each option separately. dbt Copilot saves them to the same YAML file.
```

> 3. Select the available options to generate the YAML config: **Generate Documentation**, **Generate Tests**, **Generate Semantic Model**, or **Generate Metrics**. To generate multiple YAML configs for the same model, click each option separately. dbt Copilot intelligently saves the YAML config in the same file.

Reviewer: Following the above, as i din't find the AI option, I didn't see these configs.

Reviewer: Just one more thing, with some users seeing dbt Wizard and some seeing dbt Copilot should we documenting both, maybe have 2 steps. If we’re only documenting Copilot, maybe say click the generate icon / **Generate with dbt Copilot** so people aren’t hunting for a **dbt Copilot** button?

> 3. Select the available options to generate the YAML config: **Generate Documentation**, **Generate Tests**, **Generate Semantic Model**, or **Generate Metrics**. To generate multiple YAML configs for the same model, click each option separately. dbt Copilot intelligently saves the YAML config in the same file.

Author: ```suggestion
```
